src/ts/components/charts/fragments/AreaChart.tsx

```diff
@@ -108,6 +108,9 @@ const AreaChart = ({
     return (
         <MantineAreaChart
             data-dash-is-loading={getLoadingState(loading_state) || undefined}
+            {...parseFuncProps('AreaChart', others)}
+            data={data}
+            dataKey={dataKey}
```

Comment:
Why did `data` and `dataKey` need to get added explicitly here (and in other components) rather than left in `others`? Won't they just pass through `parseFuncProps` unchanged?